package.json

```diff
@@ -28,38 +29,48 @@
 		"prettier": "prettier --write --ignore-path .gitignore \"**/*.{js,mdx,md,html,json}\"",
 		"test": "jest",
 		"storybook": "yarn workspace @lundium/storybook",
-		"create-stats": "webpack --profile --json > stats.json",
-		"analyze": "yarn create-stats && webpack-bundle-analyzer ./stats.json"
+		"theme-basic": "yarn workspace @lundium/theme-basic",
+		"analyze": "yarn create-stats && webpack-bundle-analyzer ./stats.json",
+		"create-stats": "webpack --profile --json > stats.json"
 	},
 	"devDependencies": {
-		"@babel/core": "^7.8.0",
-		"autoprefixer": "^9.7.3",
+		"@babel/core": "^7.8.3",
```

Comment:
those dependencies are not needed. babel-preset-react-union already use them. if you have troubles with setup the babel I can help. there is probably problem with `rootMode: upward` configuration.